### PR TITLE
Enable token detection for the Aurora network

### PR DESCRIPTION
**Pull request by dan437**
## Explanation
This PR: 
- Enables token detection for the Aurora network
- Fixes default currency for Aurora
- Updates a URL for dev Swaps API

- [x] This PR has a dependency on this PR in the `MetaMask/core` repo: https://github.com/MetaMask/core/pull/1327
- [x] It's also waiting for backend changes to be in prod to display token icons on the Aurora network

## Testing steps
- Have tokens with non-zero balance on the Aurora network and not displayed in the extension yet
- Click on `Refresh list` in the extension: 
image
- New tokens are detected

### CODE REVIEW

File: shared/constants/network.ts [hunk 1]
```diff
@@ -575,7 +575,7 @@ export const FEATURED_RPCS: RPCDefinition[] = [
     chainId: CHAIN_IDS.AURORA,
     nickname: AURORA_DISPLAY_NAME,
     rpcUrl: `https://aurora-mainnet.infura.io/v3/${infuraProjectId}`,
-    ticker: CURRENCY_SYMBOLS.AURORA,
```
Comment (dan437) [inline shared/constants/network.ts:578]: Default currency for Aurora is ETH, not the Aurora token: https://hackernoon.com/from-ethereum-to-aurora-with-infura

File: shared/constants/network.ts [hunk 2]
```diff
@@ -203,7 +203,7 @@ export const LOCALHOST_RPC_URL = 'http://localhost:8545';
  */
 export const CURRENCY_SYMBOLS = {
   ARBITRUM: 'ETH',
-  AURORA: 'Aurora ETH',
+  AURORA: 'AURORA ETH',
```
Comment (dan437) [inline shared/constants/network.ts:206]: Now the primary token icon loads correctly for Aurora.

**Review by mcmire — APPROVED** [review]
Yarn patch looks good to me! I can't speak to the other changes as well as I can for the Yarn patch, but they look reasonable.

**Review by brad-decker — APPROVED** [review]
LGTM

**Pull request merged**